doc/config.rst

```diff
@@ -239,4 +239,13 @@ For Seagauge waves:
 - ``calculated_wave_interval_units``: wave interval units
 - ``wp_min``, ``wp_max``: min/max allowable wave period, in seconds
 - ``wh_min``, ``wh_max``: min/max allowable wave height, in meters
-- ``wp_ratio``: maximum allowable ratio between peak period (``wp_peak``) and mean period (``wp_4060``)
\ No newline at end of file
+- ``wp_ratio``: maximum allowable ratio between peak period (``wp_peak``) and mean period (``wp_4060``)
+
+TruBlue
+---------------
```

Comment:
Please ensure you have the same number of hyphens underneath the new headings. It raises an unnecessary sphinx warning if not.

Reply:
Fixed